app/lib/system/deploy.rb

```diff
@@ -12,20 +12,25 @@ module Deploy
     # is overridden with VERSION.
     # The information is exposed via {MASTER_PORTAL}/deploy endpoint for logged-in users.
     class Info
-      VERSION = '2.x'
-      private_constant :VERSION
+      DEFAULT_VERSION = '2.x'
+      private_constant :DEFAULT_VERSION
 
       attr_reader :revision, :deployed_at, :release
 
       delegate :minor_version, :major_version, to: :version
 
       def initialize(info)
         @revision = info.fetch('revision') { `git rev-parse HEAD 2> /dev/null`.strip }
-        @release = ThreeScale.config.onpremises ? info.fetch('release', VERSION) : VERSION
+        @release = ThreeScale.config.onpremises ? info.fetch('release', DEFAULT_VERSION) : DEFAULT_VERSION
         @deployed_at = info.fetch('deployed_at') { Time.now }
         @error = info.fetch(:error) if info.key?(:error)
       end
 
+      def docs_version
+        # RHOAM version has only one segment (release = 'RHOAM')
+        @docs_version ||= release == DEFAULT_VERSION || version.segments.count < 2 ? 'red_hat_3scale/2-saas' : "red_hat_3scale_api_management/#{major_version}.#{minor_version}"
```

Comment:
This weird check `version.segments.count < 2` is intended for RHOAM, I just want to avoid hardcoding `RHOAM` (even if I did mention in in the comment)

Reply:
Yeah... maybe. But what would you suggest for the `rhoam?` method? we can of course just check for `== 'RHOAM'`, but I wanted to avoid it. But I also agree that what I did is confusing :grimacing:

Reply:
Then perhaps abstract exactly the same condition into a method called `#rolling_release?` which would indicate SaaS and ROAM. If this makes more sense.. and I'm not sure it makes more sense than `saas? || rhoam?` but would still be more understandable.

P.S. is `release == DEFAULT_VERSION` to indicate SaaS?

Reply:
> P.S. is release == DEFAULT_VERSION to indicate SaaS?

yes :sweat_smile: 

OK, I'll try to make it less confusing :upside_down_face: